apps/redi-connect/src/pages/app/find-a-mentor/FindAMentor.tsx

```diff
@@ -330,40 +328,40 @@ const FindAMentor = () => {
           </>
         )}
       </div>
-
-      <Columns>
-        {currentItems.map((mentor) => (
-          <Columns.Column
-            mobile={{ size: 12 }}
-            tablet={{ size: 6 }}
-            desktop={{ size: 4 }}
-            key={mentor.id}
-          >
-            <MentorProfileCard
-              mentorProfile={mentor}
-              linkTo={`/app/find-a-mentor/profile/${mentor.id}`}
-              toggleFavorite={() => toggleFavorite(mentor.id)}
-              isFavorite={isMentorFavorite(mentor.id)}
-            />
-          </Columns.Column>
-        ))}
-      </Columns>
-      {totalItems > MENTOR_CARDS_PER_PAGE && (
-        <Pagination
-          totalPagesNumber={totalPagesNumber}
-          currentPageNumber={currentPageNumber}
-          setCurrentPageNumber={setCurrentPageNumber}
-          scrollPosition={PAGINATION_SCROLL_POSITION}
-        />
-      )}
-
-      {mentors?.length === 0 && !isLoading && (
+      {filteredMentorProfiles?.length === 0 ? (
         <Content>
-          <>
-            Unfortunately <strong>could not find any mentors</strong> matching
-            your search criterias.
-          </>
+          Unfortunately <strong>couldn't find any mentor profiles</strong>{' '}
+          matching your search criteria. You can try adjusting your filters to
+          find more profiles.
```

Comment:
Won't this show up while the query is loading? Consider using `mentorsQuery.isLoading` (see [this page](https://tanstack.com/query/v4/docs/framework/react/guides/queries) for info on `react-query`'s status flags like `isLoading`, `isError`, `isSuccess`)

Reply:
No, it doesn't show up while the query is loading. I thought using the `?.` operator in multiple places ensured we had the data loaded, and it's enough because we are not using `queryName.isLoading` anywhere.

Reply:
Yes, you're entirely right. Then we're good to go - let's get this baby merged 🚀